config.json

```diff
@@ -86,6 +86,20 @@
         ],
         "status": "active"
       },
+      {
+        "slug": "resistor-color",
+        "name": "Resistor Color",
+        "uuid": "51c31e6a-b7ec-469d-8a28-dd821fd857d2",
+        "difficulty": 1,
+        "concepts": [
+          "external-crates"
+        ],
+        "prerequisites": [
+          "functions",
+          "enums"
+        ],
+        "status": "wip"
```

Comment:
Note that until the status is `active`, students won't see this exercise. 

The last thing that I think is necessary before setting this active is to ensure that the rust test runner has both of these dependencies in its list of supported external crates.

Reply:
> Note that until the status is active, students won't see this exercise.

Oh, right. But we can't set it to `active` yet, right? Because some files (e.g. `design.md`) are currently missing.

> The last thing that I think is necessary before setting this active is to ensure that the rust test runner has both of these dependencies in its list of supported external crates.

Is that something I can do?

Reply:
If I understood correctly, [this line](https://github.com/exercism/rust-test-runner/blob/59acc0d59da1e2c2f6778ceca2e1f9c9da4e2e79/local-registry/supported_crates#L46) and [that line](https://github.com/exercism/rust-test-runner/blob/59acc0d59da1e2c2f6778ceca2e1f9c9da4e2e79/local-registry/supported_crates#L75) indeed show that the requisite crates are supported.

Reply:
Oh very good! It's possible in principle that the versions don't line up--that's a [known issue](https://github.com/exercism/rust-test-runner/issues/37)--but I think we could switch this to active without worrying too much.

Want to do that before I merge, or would you prefer to leave it WIP for some more review first?

Reply:
I think squashing commits would be a good idea before merging (or is that something possible on your end?). But otherwise I'd leave that up to your discretion: perhaps more review is not a bad idea.

Reply:
Sure, we use the github's squash and merge by default in this repo, no worries!

In my opinion this is ready to set as active now. Worst case we have to add a PR changing it to WIP if students have problems.

Reply:
Alright, if you say it's fine, then let's do this!